data/hai/hai reveal 4 middle.txt

```diff
@@ -1548,10 +1941,48 @@ mission "Hai Reveal [A11] Sound the Alarm"
 			`	Teeneep just shakes her head, pulls out a datapad, and quickly taps a few times. "Well, that was going to happen in an hour. No point waiting, though. Let's get dinner."`
 			choice
-				`	(Get dinner.)`
+				`	"Actually, I'll go watch the summary of the proceedings."`
+				`	"Dinner sounds great to me."`
 					goto dinner
-				`	(Decline and maybe eat with the diplomats.)`
+				`	"No, thank you. I need to check in with the diplomats.`
+					goto diplomats
+			`	Teeneep gives you directions to a common area with a large holoprojector. You arrive just as the news is about to begin.`
+
+
+			# SCENE CHANGE
+
+			# Deliberation grace period is over and everybody leaves the chamber.
+
+			label news
+			`	The holoprojector turns on, and an elderly Hai presents the news. "In today's closed emergency meeting, the full Elder Council put the known disruptor, Teeneep, on trial for risking the safety of all Hai by her recent actions. While she made an eloquent defense, her previous confession was enough evidence to prove her guilt. There was audibly wide-spread support among the Elders for a stern and righteous action in defense of the secrecy that is so key to our security... until that very premise was challenged.`
+			`	"All the Council was silenced in awe as Aimet Taepip spoke in favor of Teeneep, against the secrecy doctrine, presenting a stark choice. Secrecy and safety now conflict; we must choose one or the other: Teeneep's path to safety, or Gemnali's path to secrecy. At this, some brought to attention what traditionalists have said for years: the remarkable similarity between Teeneep's life and that of Eyrisen from the Mythos. The Mythos and Aimet together convinced many Elders to back down, but not enough.`
+			`	"Today's decision... was indecision. We await another path through the chaos that slowly engulfs us. There are rumors of a petition that may reach its threshold in weeks. Perhaps an election will bring us rapidly to our salvation or doom before time runs out."`
+			`	A Hai next to you tells you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`
+			choice
+				`	"Well, you just have to keep living your life, right?"`
+				`	"I'm sure the mercenary will fix all of this."`
+				`	"Political discussions are always droll, regardless of their consequences."`
+			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time. Albeit mostly about work.`
+				goto panic
```

Comment:
The implication of this whole sequence btw, is that this is *all* meant to be restricted, non-public information - but by virtue of the mythos having been invoked, it is simply uncontainable. People moved by superstitious beliefs of fate and destiny will share those when dealing with such matters because it is fundamental to them, and so it is that power that drives the spread of information from there.
Also, alignment for briefing format.
```suggestion
			# SCENE CHANGE

			# Deliberation grace period is over and everybody leaves the chamber.

			label news
			`	The holoprojector turns on, and an elderly Hai presents. "In today's closed emergency meeting conducted in the human language, the full Elder Council put the known disruptor, Teeneep, on trial for risking the safety of all Hai by her recent actions. While she made an eloquent defense, her previous confession was considered enough evidence to prove her guilt. There was audibly wide-spread support among the Elders for a stern and righteous action in defense of the secrecy mandate... until that very premise was challenged.`
			`	"In a rare utterance Aimet Taepip silenced the Council to speak in favor of Teeneep, and against the secrecy mandate, presenting a bold reminder of recent history and a stark choice. Secrecy and safety are now conflict; we must choose one or the other. The scene inspired certain sections to invoke the mythos, 'Veeheylur and Misanaya fight over Eyrisen'. This event appears to have shaken many members of the Council who had previously declared positions. In the vote immediately following, enough Elders abstained that the matter could not be progressed.`
			`	"Today's decision... was indecision. A watch is being placed on a petition identified in association with this that may reach its threshold in weeks. An election trigger will be the outcome and all departments are on alert to take appropriate action."`
			`	"Briefing will repeat in Hai in 1 minute."`
			`	A Hai next to you tells you, "It's all a bit disconcerting, isn't it? Pirates and Unfettered on our borders and Elders doing nothing. A chaos magnet causing mischief while a human mercenary helps her. I wish I was there to see the proceedings, but I know they weren't recorded."`
			choice
				`	"Well, you just have to keep living your life, right?"`
				`	"I'm sure the mercenary will fix all of this."`
				`	"Political discussions are always droll, regardless of their consequences."`
			`	As he shrugs and walks away, you realize your acute hunger. Foraging for food in a nearby cafeteria, you soon find Alondo and Remington. They have a surprisingly civilized conversation the entire time. Albeit mostly about work.`
				goto panic
```